internal/modules/feegrant/test.py

```diff
@@ -0,0 +1,135 @@
+import time, logging
+import unittest
+from utils import exec_command, env
+from core.keys import keys_show
+from modules.feegrant.tx import *
+from modules.feegrant.query import *
+
+from modules.bank.tx import (
+    tx_send,
+)
+from modules.bank.query import (
+    query_balances,
+)
+
+DAEMON_HOME = env.DAEMON_HOME
+NODE2_HOME = env.get("NODE2_HOME")
+
+# get account addresses
+granter = keys_show("validator1")[1]["address"]
+grantee_1 = keys_show("account1")[1]["address"]
+grantee_2 = keys_show("account2")[1]["address"]
+receiver = keys_show("validator2", "acc", NODE2_HOME)[1]["address"]
+
+amount = 5
+fees = 2
+
+
+class TestFeegrantModuleTxsQueries(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # set periodic grant by passing granter key and grantee address
+        status, response = set_periodic_grant("validator1", grantee_2)
+        assert status
+        time.sleep(3)
+
+    def test_granter_as_fee_payer(self):
+        # grant tx
+        status, response = tx_grant("validator1", grantee_1)
+        assert status
```

Comment:
Let's keep the consistency
```suggestion
        self.assertTrue(status)
```